### PR TITLE
Improve usability of the quick run button

### PR DESCRIPTION
**Pull request by znicholasbrown**
PR Checklist:

- [ ] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

## Describe this PR
This PR removes the automatic routing of the quick run button and introduces a new toast system for stackable global toasts with a straightforward CRUD system. It should now be a lot easier to create a bunch of runs with default parameters!

https://user-images.githubusercontent.com/27291717/118725558-19f51380-b7fe-11eb-8692-6a220b90c261.mov

### CODE REVIEW

**Comment by madkinsz** [comment]
Sweet. I hit this exact pain point just a couple days ago.

**Review by ThatGalNatalie — APPROVED** [review]
lgtm!

File: src/pages/Flow/Actions.vue [hunk 1]
```diff
@@ -117,45 +128,58 @@ export default {
...
-        this.quickRunErrorAlert()
+        await this.updateNotification({
+          id: notificationId,
+          notification: {
+            color: 'error',
+            loading: false,
+            text: 'There was a problem creating your run',
+            subtext: err?.message?.includes('archived')
+              ? 'This flow version is archived - you likely have a newer version of your flow'
+              : 'Error',
```
Comment (zhen0) [inline src/pages/Flow/Actions.vue:174]: Maybe this is one for wider discussion but I prefer adding the error message passed back by the API.  They're now written in a way that is meant to be user friendly and help us and users with de-bugging.  Either way, I'm not sure 'Error' is helpful - if we don't include the actual error message maybe just leave blank?

**Review by zhen0 — COMMENTED** [review]
This looks really good @znicholasbrown - one comment about error messages.

File: src/pages/Flow/Actions.vue (hunk 1 above)
Comment (znicholasbrown) [reply src/pages/Flow/Actions.vue:174]: Yeah I agree in most cases wrt API error messages but less so in application errors that we don't explicitly throw. In this particular instance though, the API is returning almost the same message but with an id embedded, which isn't necessary to show the user (they're already on the flow page!)

**Review by zhen0 — APPROVED** [review]
LGTM @znicholasbrown

**Pull request merged**